src/packages/@deprecate/components/environments/environments-list/sub-components/environment-tab/EnvironmentTab.svelte

```diff
@@ -191,7 +190,7 @@
   isOpen={isEnvironmentPopup}
   handleModalState={handleEnvironmentPopUpCancel}
 >
-  <div class="text-lightGray mb-1 sparrow-fs-14">
+  <div class="mb-1 sparrow-fs-14" style="color: #ccc;">
```

Comment:
Rather than using `style` attribute and hardcoding the color, can we give color a name and use it using `class` attribute itself? This will ensure in future when we reuse the same color, we don't have to hard code again. And keep in mind, in future we may want to support dark mode and light mode both. In these cases, hardcoding colors is not a good idea.

In fact, the class names should not also directly point to colors verbally. We should talk about shades.

Example, `accentColor`, `accentColorDark`, `accentColorDarker`, `accentColorDarkest` etc. This is just one of the convention, we could use a different one as well.